cli/pcluster/config/validators.py

```diff
@@ -782,6 +782,27 @@ def s3_bucket_validator(param_key, param_value, pcluster_config):
 
     return errors, warnings
 
+def fsx_lustre_auto_import_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    if param_value != "NONE":
+        try:
+            fsx_section = pcluster_config.get_section("fsx")
+            fsx_import_path = fsx_section.get_param_value("import_path")
+            bucket = getBucketNameFromImportPath(fsx_import_path)
+            s3BucketRegion = boto3.client("s3").get_bucket_location(Bucket=bucket)["LocationConstraint"]
+            # Buckets in Region us-east-1 have a LocationConstraint of null
+            if (s3BucketRegion is None):
+                s3BucketRegion = "us-east-1"
+            if s3BucketRegion != pcluster_config.region:
+                errors.append("AutoImport is not supported for cross-region buckets.")
+        except ClientError:
```

Comment:
A couple of questions for how this exception is handled:
* Is there a set of a ClientError's attributes that could be checked to be more certain about whether the bucket doesn't exist or the user doesn't have access?
* warnings are used when we want to bring an issue to the user's attention, but we don't expect the issue to cause cluster creation failures. this seems like something that should be added to errors, because I would expect file system creation to fail.